quantum/velocikey.h

```diff
@@ -0,0 +1,13 @@
+#ifndef VELOCIKEY_H
+#define VELOCIKEY_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+bool velocikey_enabled(void);
+void velocikey_toggle(void);
+void velocikey_accelerate(void);
+void velocikey_decelerate(void);
+uint8_t velocikey_match_speed(uint8_t minValue, uint8_t maxValue);
+
```

Comment:
```suggestion
uint8_t interval_time;
```

Reply:
`interval_time` will need to be defined in `rgblight.c` since this header is only conditionally included, whereas `interval_time` is always used.